frontend/src/pages/projects/components/PasswordInput.tsx

```diff
@@ -8,7 +8,11 @@ const PasswordInput: React.FC<React.ComponentProps<typeof TextInput>> = (props)
   return (
     <InputGroup>
       <TextInput {...props} type={isPassword ? 'password' : 'text'} />
-      <Button variant="control" onClick={() => setPassword(!isPassword)}>
+      <Button
+        aria-label={isPassword ? 'Show key' : 'Hide key'}
```

Comment:
hmmm -- maybe this should be something like `Show secret`, `Show text`, `Show hidden text`, or `Show password`? `Key` seems unclear. Not sure if I would be able to understand that if I just heard a screen reader show me. Thoughts?

Reply:
Agreed - updated to `show/hide password` which would make sense as this component is called `PasswordInput`.